securedrop_client/queue.py

```diff
@@ -28,6 +28,38 @@
 logger = logging.getLogger(__name__)
 
 
+class RunnablePriorityQueue(PriorityQueue):
+    """
+    Wrapper class around PriorityQueue that emits a signal when message or reply
+    download jobs are enqueued or dequeued.
+    """
+
+    def __init__(
+        self, *args: Any, queue_updated_signal: Optional[pyqtSignal] = None, **kwargs: Any
```

Comment:
Given https://github.com/freedomofpress/securedrop-client/pull/1604#discussion_r1047896459, this should be `pyqtBoundSignal`. (We're never manipulating unbound signals that I know of, they're always bound to an instance of the class where we define them.)

The type can be imported from `PyQt5.QtCore`.

```suggestion
        self, *args: Any, queue_updated_signal: Optional[pyqtBoundSignal] = None, **kwargs: Any
```

Reply:
Don't worry about other occurrences that you didn't introduce, we'll fix all those at once in #1611.